src/Plazas/ReservasList.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { Button, ButtonGroup, Container, Table } from 'reactstrap';
+import AppNavbar from '../AppNavBar';
+import { Link } from 'react-router-dom'
+import Navbar from '../components/Navbar'
+import ListComponent from '../components/ListComponent'
+class ReservasList extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {reservas: []};
+       // this.remove = this.remove.bind(this);
+    }
+
+    componentDidMount() {
+        fetch('https://park-inn-ispp-be.herokuapp.com/propietarios/reservas/all')
```

Comment:
sustituir URL por 'https://park-inn-ispp-be.herokuapp.com/reservas/usuario/1'

Reply:
Sustituido